rules/windows/credential_access_copy_ntds_sam_volshadowcp_cmdline.toml

```
@@ -25,9 +28,9 @@ type = "eql"
 
 query = '''
 process where event.type in ("start", "process_started") and
- process.pe.original_file_name in ("Cmd.Exe", "PowerShell.EXE", "XCOPY.EXE") and
- process.args : ("copy", "xcopy", "Copy-Item", "move", "cp", "mv") and
- process.args : ("*\\ntds.dit", "*\\config\\SAM", "\\*\\GLOBALROOT\\Device\\HarddiskVolumeShadowCopy*\\*")
+ process.pe.original_file_name in ("Cmd.Exe", "PowerShell.EXE", "XCOPY.EXE", "esentutl.exe") and
+ process.args : ("copy", "xcopy", "Copy-Item", "move", "cp", "mv", "*/y*", "*/vss*", "*/d*") and
+ process.args : ("*\\ntds.dit", "*\\config\\SAM", "\\*\\GLOBALROOT\\Device\\HarddiskVolumeShadowCopy*\\*", "*/system32/config/SAM*", "*/SAM*")
```

We should keep the associated args and processes  separate to be more specific with the logic. Even though someone running them may be nonsensical (like `esentutil.exe copy-item ...`) it is still good practice to make the logic specific and obvious for readability down the road.

```suggestion
  (
    (process.pe.original_file_name in ("Cmd.Exe", "PowerShell.EXE", "XCOPY.EXE") and
       process.args : ("copy", "xcopy", "Copy-Item", "move", "cp", "mv")
    ) or
    (process.pe.original_file_name : "esentutl.exe" and process.args : ("*/y*", "*/vss*", "*/d*"))
  ) and
  process.args : ("*\\ntds.dit", "*\\config\\SAM", "\\*\\GLOBALROOT\\Device\\HarddiskVolumeShadowCopy*\\*", "*/system32/config/SAM*", "*/SAM*")
```

---

Any way to tighten down `"*/SAM*"`?

Also, that would include `"*/system32/config/SAM*"`, making it redundant

---

Yeah  agree.